### PR TITLE
feature(ImageViewer): Add FilterControls component to ImageViewer

### PR DESCRIPTION
**Pull request by mheitman**
to: @williaster @alecklandgraf

## Description
This adds filters to the `ImageViewer` component to adjust image brightness and contrast to complement the existing `RotateControls` and `ZoomControls`.

## Motivation and Context
This will be used to further enhance image editing capabilities in the `ImageViewer`. 

## Testing
Local storybook

## Screenshots 
High brightness:

image

Low brightness:

image

## Checklist

- My code follows the style guide of this project.
- I have updated or added documentation accordingly.
- I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by mheitman** [comment]
@williaster @alecklandgraf PTAL 🙂

File: packages/core/src/components/ImageViewer/FilterControls.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+import IconButton from '../IconButton';
+import Range from '../Range';
+import Text from '../Text';
+import T from '../Translate';
+import useStyles, { StyleSheet } from '../../hooks/useStyles';
+
+export type FilterControlsProps = {
+  /** The current brightness. 1 by default. */
+  brightness?: number;
+  /** Callback when brightness changes */
```
Comment (williaster) [inline packages/core/src/components/ImageViewer/FilterControls.tsx:15]: missing `.`

File: packages/core/src/components/ImageViewer/FilterControls.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  onBrightnessChange: (brightness: number) => void;
+  /** The current contrast. 1 by default. */
+  contrast?: number;
+  /** Callback when contrast changes */
```
Comment (williaster) [inline packages/core/src/components/ImageViewer/FilterControls.tsx:19]: missing `.`

File: packages/core/src/components/ImageViewer/FilterControls.tsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  onContrastChange: (contrast: number) => void;
+  /** Size of the icons. */
+  iconSize?: number | string;
+  /** Place dropdown menu above */
```
Comment (williaster) [inline packages/core/src/components/ImageViewer/FilterControls.tsx:23]: missing `.`

File: packages/core/src/components/ImageViewer/FilterControls.tsx [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+import Card, { Content } from '../Card';
+import Dropdown from '../Dropdown';
+import IconButton from '../IconButton';
+import Range from '../Range';
+import Text from '../Text';
+import T from '../Translate';
+import useStyles, { StyleSheet } from '../../hooks/useStyles';
+
+export type FilterControlsProps = {
+  /** The current brightness. 1 by default. */
```
Comment (williaster) [inline packages/core/src/components/ImageViewer/FilterControls.tsx:13]: might be worth mentioning what the valid range of `brightness` values is?

File: packages/core/src/components/ImageViewer/FilterControls.tsx [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+  brightness?: number;
+  /** Callback when brightness changes */
+  onBrightnessChange: (brightness: number) => void;
+  /** The current contrast. 1 by default. */
```
Comment (williaster) [inline packages/core/src/components/ImageViewer/FilterControls.tsx:17]: might be worth mentioning what the valid range of `contrast` values is?

**Review by williaster — COMMENTED** [review]
Looks great overall, had a couple minor comments on prop annotations

**Comment by mheitman** [comment]
@williaster updated the comments -- PTAL!

**Review by williaster — APPROVED** [review]

**Pull request merged**